timecog/timecog.py

```diff
@@ -136,8 +136,8 @@ async def remindme(self, ctx, *, time):
         else:
             for ir in time_in_regeces:
                 match = re.search(ir, time, re.IGNORECASE)
-                if not match:
-                    break # Only use the first one
+                if not match: # Only use the first one
```

Comment:
This is a bit weird. Why not just match against time_in_regeces[0] instead of the loop?